GOVERNANCE.md

```diff
@@ -42,9 +45,40 @@ Maintainers have right to nominate contributors to become maintainers and vote o
 
```

Comment:
> The nomination is done as a pull request to the CODEOWNERS file in corresponding project's git repository. The approval must be done on the pull request by project's maintainers.

I am sorry I did not spot this earlier. It is kind of hidden unless you expand the file...

I like this on a ideology standpoint, deciding this in a PR. HOWEVER, this is not how maintainers are represented in github. The same maintainer group can be responsible for several repositories and we have a separate list in github for maintainers. If we would use the CODEOWNERS file to keep track of maintainers there will be divergence between the maintainer list as it is represented in the github gui and the CODEOWNERS file, and this is not so good in my opinion.

We need some other method to appoint maintainers.

Just for the record, I don't know what privileges CODEOWNERS have in a github project as I have not explored or come in contact with that before.

Reply:
We need to look at this. Thanks for highlighting.